### DISCUSSION TITLE
issue on safari

### DISCUSSION THREAD

**Discussion by nicodioso** [Akryum/vue-observe-visibility#26] [Q&A]
my version of safari is 11.1.1

used v-observe-visibility="(isVisible, entry) => {animateIntro(`pop-up`, isVisible, entry)}" as html attribute

<img width="1280" alt="screen shot 2018-07-25 at 11 37 23 am" src="https://user-images.githubusercontent.com/40595899/43178369-aed4dc1c-8fff-11e8-8146-45dd8676f813.png">

**Comment by ofek-a** [comment]
I had the same issue.

Before I apparently had my polyfill loaded after the initialization of the plugin, which I believe resulted in the error.

I resolved it by simply adding the polyfill from polyfill.io in my head section of the HTML, like that:
```html
  <script charset="utf-8" src="https://cdn.polyfill.io/v2/polyfill.min.js?features=IntersectionObserver,IntersectionObserverEntry"></script>
```
